react/TabbedShelf/tabbedShelf.css

```diff
@@ -0,0 +1,77 @@
+.container {
```

Comment:
Add these handlers into `shelf.css`. I prefer to maintain only one CSS file to maintain.

Reply:
me and bebber think it is better to leave like this. These are styles only used at the TabbedShelf unique components